minibroker/scripts/generate-minibroker-repo.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+set -e
+set -x
+cp s3.*chart/*.tgz repo
+helm repo index --url https://minibroker-helm-charts.s3.amazonaws.com/minibroker-charts repo
```

Comment:
Isn't the s3 bucket name supposed to be customizable?

Reply:
Yes. Fixed in 1ea3ce7. This file isn't needed any more, there's a generic one that handles the both the minibroker and the customized helm charts.